js/conditionals.js

```diff
@@ -0,0 +1,214 @@
+"use strict";
+
+/**
+ * TODO:
+ * Write some JavaScript that uses a `confirm` dialog to ask the user if they
+ * would like to enter a number. If they click 'Ok', prompt the user for a
+ * number, then use 3 separate alerts to tell the user:
+ *
+ * - whether the number is even or odd
+ * - what the number plus 100 is
+ * - if the number is negative or positive
+ *
+ * if what the user enters is not a number, use an alert to tell them that, and
+ * do *not* display any of the above information.
+ *
+ * Can you refactor your code to use functions?
+ */
+
+function messageForEvenOrOdd(number) {
+    var message;
+    if (number % 2 === 0) {
+        message = "This is an even number!";
+    } else {
+        message = "This is an odd number!";
+    }
+    return message;
+}
+
+function plusOneHundMessage(number) {
+    return "This number plus 100 is " + (number + 100) + "!";
+}
+function mathGame() {
+    var go = confirm("Hey, want to do some math?");
+    if (go === true) {
+        var number = parseInt(prompt("Give me a number"));
+        if (isNaN(number)) {
+            alert("You did not enter a number.")
+        } else {
+            alert(messageForEvenOrOdd(number));
+            alert(plusOneHundMessage(number));
+            if (number >= 0) {
+                alert("This number is positive!")
+            } else {
+                alert("This number is negative!")
+            }
+        }
+    }
+    return number;
+}
+
+// mathGame();
+
+/* ########################################################################## */
+
+/**
+ * TODO:
+ * Create a function named `analyzeColor` that accepts a string that is a color
+ * name as input. This function should return a message that related to that
+ * color. Only worry about the colors defined below, if the color passed is not
+ * one of the ones defined below, return a message that says so
+ *
+ * Example:
+ *  > analyzeColor('blue') // returns "blue is the color of the sky"
+ *  > analyzeColor('red') // returns "Strawberries are red"
+ *  > analyzeColor('cyan') // returns "I don't know anything about cyan"
+ *
+ * You should use an if-else-if-else block to return different messages.
+ *
+ * Test your function by passing various string literals to it and
+ * console.logging the function's return value
+ */
+
+function analyzeColor(color) {
+    var message;
```

Comment:
It may be less work to do `color = color.toLowerCase()` before the conditional structures. That way, you've got one line that does the job of multiple lines 🗄

Reply:
I have set `color = color.toLowerCase()` before beginning to navigate the conditions in `analyzeColor()` and `analyzeColorSwitch()`